src/main/java/core/basesyntax/FigureDraw.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface FigureDraw {
```

Comment:
The interface name `FigureDraw` is a verb phrase, which is not recommended for class/interface names. Consider renaming it to something that represents an entity or a capability like `Drawable` or `FigureDrawable`, which would be more in line with Java naming conventions and the checklist item regarding class/interface names.